tests/plugins/test_cachi2_init.py

```diff
@@ -370,6 +372,35 @@ def test_dependency_replacements(workflow):
                          expect_error="Dependency replacements are not supported by Cachi2")
 
 
+def test_enforce_sandbox(workflow: DockerBuildWorkflow, tmp_path: Path) -> None:
+    """Should remove symlink pointing outside of repository"""
+    container_yaml_config = dedent(f"""\
+            remote_source:
+              flags:
+                - remove-unsafe-symlinks
+              repo: {REMOTE_SOURCE_REPO}
+              ref: {REMOTE_SOURCE_REF}
+              pkg_managers: []
+            """)
+
+    mock_repo_config(workflow, data=container_yaml_config)
+
+    def clone_f(repo, target_dir, ref):
+        bad_symlink = Path(target_dir / 'symlink_to_root')
+        with open(target_dir / "clone.txt", "w") as f:
+            f.write(f"{repo}:{ref}")
+            f.flush()
+        write_file_tree({bad_symlink: Symlink("/")}, target_dir)
+        assert Path(target_dir / bad_symlink).exists()
+
+    mocked = flexmock(Cachi2InitPlugin)
+    mocked.should_receive('clone_remote_source').replace_with(clone_f)
+    Cachi2InitPlugin(workflow).run()
+
+    # We have no way to know the exact path ahead of time
```

Comment:
Path should be stored in workflow

Reply:
https://github.com/containerbuildsystem/atomic-reactor/blob/0f26fe21d34673108a8a20642e8e45c68df58235/tests/plugins/test_cachi2_init.py#L107 something like this

Reply:
Fixed